inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go.  This document formalizes a Hyperledger wide polciy for archiving projects and moving maintainers to emereitus status when a proejct does not have an explicit policy of their own.
+
+## Maintainer Inactivity
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privledges is in the interest of an orderly and secure project.
+
+Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be removed from the relevant Github Organizations.
```

Comment:
```suggestion
Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be removed from the relevant Github teams.
```

Reply:
What counts as a contribution here? I hope this is not limited to PRs and commits. Reviewing and merging PRs is an important part of the maintainers role that we ought to recognize even if that's they have done.

Reply:
@tkuhrt removing them from a team is hard - binning contributions by role is not reliable.

Reply:
@lehors [events that show up in the audit log](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization#audit-log-actions) are what we can gate on. There are a lot of activities here - some make sense, some don't.

Reply:
> @tkuhrt removing them from a team is hard - binning contributions by role is not reliable.

@ryjones : I was thinking that if a maintainer is part of a project-maintainer team, then we would want to remove them from that. Are you saying that the preferred mechanism would be to drop them completely from the organization? What does that do for GitHub teams they were previously part of? Does it automatically remove them from those teams?

Reply:
@tkuhrt you're right, I conflated two discussions :)